src/server/server.js

```diff
@@ -47,6 +48,11 @@ azure
         process.exit(1);
     });
 
+stsclient.init(config.nav);
+stsclient.hentAccessToken().catch(err => {
```

Comment:
Å hente token for cache-formål er en implementasjonsdetalj som ikke trenger å lekke ut av stsclient, bør gjøres i stsclient.init(). Ellers 👍

Reply:
Ah, riktig. Jeg tenkte vel at server.js fint kan være involvert i oppstartsaktiviteter for appen, men jeg kan endre på det 👍